packages/client-interaction/runtime/directive.ts

```diff
@@ -9,8 +9,20 @@ const firstInteraction = new Promise<void>(resolve => {
     }
 })
 
-export default (async load => {
+export default (async (load, opts) => {
     await firstInteraction
-    const hydrate = await load()
-    await hydrate()
+    const hy = async () => {
+        const hydrate = await load()
+        await hydrate()
+    }
+    const arrOpts = Array.isArray(opts.value) ? opts.value : [opts.value]
```

Comment:
`opts.value` being an array does not align with its typescript type. Why was this necessary?

Reply:
Options can be set like `client:context={['data:categories','data:grids']}`, but anyway here it would not be necessary, maybe just to avoid the need for a breaking change in a future with another option wanted for this `client:interaction` directive

Reply:
I don't think a breaking change will be necessary. We can introduce another value type anytime, like we are already doing by allowing a string now.

Reply:
You're right, my bad...